saga/src/main/java/io/axoniq/dev/samples/payment/api/SagaEndedEvent.java

```diff
@@ -0,0 +1,16 @@
+package io.axoniq.dev.samples.payment.api;
+
+import io.axoniq.dev.samples.uuid.OrderId;
+
+public class SagaEndedEvent {
```

Comment:
I think this name is a little to generic for the sample. I wouldn't want people to think that `SagaEndedEvent` is a fine name for an event in most cases, for example :sweat_smile: